bot/exts/evergreen/trivia_quiz/_cog.py

```diff
@@ -2,47 +2,91 @@
 import json
 import logging
 import random
+import typing as t
 from pathlib import Path
 
 import discord
-from discord.ext import commands
+from discord.ext import commands, tasks
 from fuzzywuzzy import fuzz
 
-from bot.constants import Roles
-
+from bot import constants
+from bot.bot import Bot
+from bot.exts.evergreen.trivia_quiz import _helpers
 
 logger = logging.getLogger(__name__)
 
+QUESTION_LIMIT = 4
+HINTS_LIMIT = 2
+
+WRONG_ANS_RESPONSE = (
+    "No one answered correctly!", "Better luck next time"
+)
+WINNER_RESPONSE = (
+    "Congratulations {winners_mention} :tada: "
+    "{winner_word} have won this quiz game with a grand total of {highest_points} points!"
+)
+
+RULES = {
+    1: "No cheating and have fun!",
+    2: "Points for each question reduces by 25 after 10s or after a hint. Total time is 30s per question"
+}
+CATEGORIES = {
+    "general": "Test your general knowledge",
+    "wikipedia": "Guess the title of random wikipedia passages."
+    # "retro": "Questions related to retro gaming."
+}
 
-WRONG_ANS_RESPONSE = [
-    "No one answered correctly!",
-    "Better luck next time"
-]
+RANDOM_WIKI_URL = "https://en.wikipedia.org/api/rest_v1/page/random/summary"
+
+p = Path("bot", "resources", "evergreen", "trivia_quiz.json")
+with p.open(encoding="utf8") as json_data:
+    QUESTIONS = json.load(json_data)
 
 
 class TriviaQuiz(commands.Cog):
     """A cog for all quiz commands."""
 
-    def __init__(self, bot: commands.Bot) -> None:
+    def __init__(self, bot: Bot) -> None:
         self.bot = bot
-        self.questions = self.load_questions()
+
+        self.wiki_questions: t.List = []
+
         self.game_status = {}  # A variable to store the game status: either running or not running.
         self.game_owners = {}  # A variable to store the person's ID who started the quiz game in a channel.
-        self.question_limit = 4
         self.player_scores = {}  # A variable to store all player's scores for a bot session.
         self.game_player_scores = {}  # A variable to store temporary game player's scores.
```

Comment:
So currently, we are using 4 variables to store the state of a quiz game.
Would it be better to have a class representing a game? like 
```py
class Game:
    def __init__(self, author, *args *kwargs):
        ...
    
```

and then in the Cog init method, 
we can have a variable `games` (a dict).
the `games` dict would look something like this
```py
{
channel_id: Game(...) class instance,
}
```

Also, another question is if we still want to store game instances, because this was done as the plan was to have this command run in multiple channels.
If the bot is restricted to only 1 channel, then it could become simpler.